releases/release-5.3.1.md

```diff
@@ -0,0 +1,215 @@
+        (dup) - Fix the negative value error in the changefeed checkpoint lag [#3010](https://github.com/pingcap/tiflow/issues/3010)
+        - Fix HTTP API not working when there are TiCDC nodes of different versions in one cdc cluster. [#3483](https://github.com/pingcap/tiflow/issues/3483)
+        - Fix a bug when too often query status is called in Load unit, DM-worker may encounter data race or panic [#3457](https://github.com/pingcap/tiflow/issues/3457)
+        - Fix OOM in container environments. [#3439](https://github.com/pingcap/tiflow/pull/3439)
+        - Please add a release note. If you don't think this PR needs a release note then fill it with `None`. [#3868](https://github.com/pingcap/tiflow/pull/3868)
+        - Please add a release note. If you don't think this PR needs a release note then fill it with `None`. [#3256](https://github.com/pingcap/tiflow/pull/3256)
+
+## Bug Fixes
+
++ TiDB
+
+    - Planner: fix wrong range calculation for Nulleq function on Enum values [#32428](https://github.com/pingcap/tidb/issues/32428)
+    (dup) - Fix the issue that INDEX HASH JOIN returns the `send on closed channel` error [#31129](https://github.com/pingcap/tidb/issues/31129)
+    (dup) - Fix the issue that some checks are skipped when TiDB Lightning is restarted [#30772](https://github.com/pingcap/tidb/issues/30772)
+    (dup) - Fix the issue of wrong import result that occurs when TiDB Lightning does not have the privilege to access the `mysql.tidb` table [#31088](https://github.com/pingcap/tidb/issues/31088)
+    (dup) - Fix the issue that concurrent column type change causes inconsistency between the schema and the data [#31048](https://github.com/pingcap/tidb/issues/31048)
+    - Add a new config to control whether support incremental import. [#27919](https://github.com/pingcap/tidb/issues/27919)
+    - Executor: fix pipelined window invalid memory address [#30326](https://github.com/pingcap/tidb/issues/30326)
+    (dup) - Fix the issue that window functions might return different results when using a transaction or not [#29947](https://github.com/pingcap/tidb/issues/29947)
+    (dup) - Fix the issue that the SQL statements that contain `cast(integer as char) union string` return wrong results [#29513](https://github.com/pingcap/tidb/issues/29513)
+    (dup) - Fix the issue that the length information is wrong when casting `Decimal` to `String` [#29417](https://github.com/pingcap/tidb/issues/29417)
+
++ TiFlash
+
+    - Align unix_timestamp behavior with TiDB and mysql when input is earlier than 1970-01-01 00:00:01 UTC
+
++ PD
+
+    (dup) - Fix a bug that the schedule generated by the region scatterer might decrease the number of peers [#4565](https://github.com/tikv/pd/issues/4565)
+    - Fix incomplete replicate file [#4328](https://github.com/tikv/pd/issues/4328)
+    (dup) - Fix a panic issue that occurs after the TiKV node is removed [#4344](https://github.com/tikv/pd/issues/4344)
```

Comment:
+ Tools
    - Backup & Restore (BR)

    - TiCDC
        - Fix a bug that long varchar will report error of "Column length too big" [#4637](https://github.com/pingcap/tiflow/issues/4637)
        - Fix a bug that owner exits abnormally when PD leader is killed [#4248](https://github.com/pingcap/tiflow/issues/4248)
        - Fix the issue that update statement execute error in safemode may cause DM-worker panic [#4317](https://github.com/pingcap/tiflow/issues/4317)
        - Fix kv client cached region metric could be negative [#4290](https://github.com/pingcap/tiflow/pull/4290)
        - Stopping tasks during load phase won't cause the source to be transfered [#3771](https://github.com/pingcap/tiflow/issues/3771)
        - Correct query-staus progress for loader [#3252](https://github.com/pingcap/tiflow/issues/3252)
        - Fix the bug that http API panics when the required processor info is not exist [#3840](https://github.com/pingcap/tiflow/issues/3840)
        - Manage sink checkpoint per table to avoid checkpoint timestamp advance unexpected [#4083](https://github.com/pingcap/tiflow/pull/4083)
        - Fix a bug that when master and worker restart in a particular order, relay status in DM-master is wrong [#3478](https://github.com/pingcap/tiflow/issues/3478)
        - Fix a bug that DM-worker can't boot up after restart [#3344](https://github.com/pingcap/tiflow/issues/3344)
        - Fix a bug that DM task will failed when PARTITION DDL cost a long time [#3854](https://github.com/pingcap/tiflow/issues/3854)
        - Fix a bug that DM may report "invalid sequence" when upstream is MySQL 8.0 [#3847](https://github.com/pingcap/tiflow/issues/3847)
        - Fix a bug that redo logs are not cleaned up when removing a paused changefeed. [#3919](https://github.com/pingcap/tiflow/pull/3919)
        - CDC client works now when cert's common name was not specified [#3882](https://github.com/pingcap/tiflow/pull/3882)
        - Add pre clean up process when s3 enable [#3878](https://github.com/pingcap/tiflow/issues/3523)
        - Fix a bug of data loss when DM does finer grained retry [#3487](https://github.com/pingcap/tiflow/issues/3487)
        - HTTP API works as expected now if there are TiCDC nodes of different versions in one cdc cluster. [#3483](https://github.com/pingcap/tiflow/issues/3483)

    - Dumpling

    - TiDB Binlog

    - TiDB Lightning